src/Symfony/Component/Security/Core/Authorization/TraceableAccessDecisionManager.php

```diff
@@ -81,7 +88,7 @@ public function getStrategy()
     }
 
     /**
-     * @return array
+     * @return iterable
```

Comment:
technically, this is a BC break if the calling code passes it to a function requiring an array